tests/integration/frameworks/wordpress/test_wordpress_do_action.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should properly instrument Wordpress do_action hooks.
+*/
+
+/*SKIPIF
+<?php
+*/
+
+/*INI
+newrelic.framework = wordpress
+*/
+
+/*EXPECT
+f
+g
+*/
+
+/*EXPECT_METRICS
+[
+  "?? agent run id",
+  "?? start time",
+  "?? stop time",
+  [
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/all"},      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/allOther"}, [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Framework/WordPress/Hook/f"},                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Framework/WordPress/Hook/g"},                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/all"},                                [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/php__FILE__"},                        [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime"},                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime/php__FILE__"},               [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Forwarding/PHP/enabled"},       [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Metrics/PHP/enabled"},          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/framework/WordPress/forced"},           [1, "??", "??", "??", "??", "??"]]
+  ]
+]
+*/
+
+// Simple mock of wordpress's do_action()
+function do_action($tag, ...$args) {
+    call_user_func_array($tag, $args);
+}
+
+function g() {
+    echo "g\n";
+}
+
+function f() {
+    echo "f\n";
+    do_action("g");
+}
+
+do_action("f");
```

Comment:
possible to nest and throw exception?

Reply:
added in [79c7ee8](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/79c7ee84f4d33549eb345c94afaea0a58874d33d)